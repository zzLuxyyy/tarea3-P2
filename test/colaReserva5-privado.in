crearColaReservas
crearSocio 23456789 Lucia Perez 10 03 2010
crearFecha 05/06/1967
crearLibro 843760494 "Cien años de soledad" "Gabriel" "García Márquez" "La historia de la familia Buendía" 3
crearReserva
encolarColaReservas
crearSocio 12345678 Jim Dear 10 03 2004
crearFecha 01/01/1961
crearLibro 421233446 "La ratesa" "Günter" "Grass" "Una alegoría postnuclear narrada por una rata" 2
crearReserva
encolarColaReservas
imprimirColaReservas
cantidadColaReservas
desencolarColaReservas
imprimirColaReservas
frenteColaReservas
cantidadColaReservas
crearSocio 45678901 Juan Diaz 10 03 1999
crearFecha 01/01/1968
crearLibro 431244557 "Lo bello y lo triste" "Yasunari" "Kawabata" "Un escritor se reencuentra con una amante" 1
crearReserva
encolarColaReservas
crearSocio 56789012 Pedro Pascal 15 08 1975
crearFecha 01/01/1966
crearLibro 411222335 "Todos los fuegos el fuego" "Julio" "Cortázar" "Ocho cuentos que combinan lo cotidiano" 1
crearReserva
encolarColaReservas
imprimirColaReservas
cantidadColaReservas
desencolarColaReservas
desencolarColaReservas
imprimirColaReservas
frenteColaReservas
cantidadColaReservas
liberarColaReservas
Fin
