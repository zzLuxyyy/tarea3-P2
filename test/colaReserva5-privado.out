1>Se ejecutó 'crearTColaReservas' exitosamente
2>Socio de ci 23456789 creado exitosamente.
3>4>5>Reserva de libro de isbn 843760494 a socio 23456789 creada exitosamente.
6>Se ejecutó 'encolarTColaReservas' exitosamente.
7>Socio de ci 12345678 creado exitosamente.
8>9>10>Reserva de libro de isbn 421233446 a socio 12345678 creada exitosamente.
11>Se ejecutó 'encolarTColaReservas' exitosamente.
12>Cola de Reservas:
Reserva de libro Cien años de soledad por Lucia Perez.
Reserva de libro La ratesa por Jim Dear.
13>La cantidad de reservas en la cola es 2
14>Se ejecutó 'desencolarTColaReservas' exitosamente
15>Cola de Reservas:
Reserva de libro La ratesa por Jim Dear.
16>Reserva obtenida al ejecutar frente:
Reserva de libro La ratesa por Jim Dear.
17>La cantidad de reservas en la cola es 1
18>Socio de ci 45678901 creado exitosamente.
19>20>21>Reserva de libro de isbn 431244557 a socio 45678901 creada exitosamente.
22>Se ejecutó 'encolarTColaReservas' exitosamente.
23>Socio de ci 56789012 creado exitosamente.
24>25>26>Reserva de libro de isbn 411222335 a socio 56789012 creada exitosamente.
27>Se ejecutó 'encolarTColaReservas' exitosamente.
28>Cola de Reservas:
Reserva de libro La ratesa por Jim Dear.
Reserva de libro Lo bello y lo triste por Juan Diaz.
Reserva de libro Todos los fuegos el fuego por Pedro Pascal.
29>La cantidad de reservas en la cola es 3
30>Se ejecutó 'desencolarTColaReservas' exitosamente
31>Se ejecutó 'desencolarTColaReservas' exitosamente
32>Cola de Reservas:
Reserva de libro Todos los fuegos el fuego por Pedro Pascal.
33>Reserva obtenida al ejecutar frente:
Reserva de libro Todos los fuegos el fuego por Pedro Pascal.
34>La cantidad de reservas en la cola es 1
35>Se ejecutó 'liberarTColaReservas' exitosamente
36>Fin.
